src/azul/transformer.py

```diff
@@ -269,6 +295,12 @@ class Contribution(Document):
     bundle_version: BundleVersion
     bundle_deleted: bool
 
+    def __post_init__(self):
+        # Cannot provide a default for version_type at the class level due to
+        # limitations with @dataclass. Its value may change to VersionType.none
```

Comment:
I was initially confused by this comment because I thought the two sentences were related. 

```suggestion
        # limitations with @dataclass.
        # May change to VersionType.none
```

Reply:
I see your point. I'll break it up.